src/lib/viewers/error/PreviewErrorViewer.js

```diff
@@ -76,23 +80,12 @@ class PreviewErrorViewer extends BaseViewer {
 
         const { displayMessage, details, message } = error;
         const { file } = this.options;
+        const { extension } = file || {};
+        const iconExtension = extension === 'flv' || extension === 'tgz' || extension === 'zip' ? extension : undefined;
```

Comment:
Is the `zip` condition still needed?

Reply:
Good question. I left it as-is from the prior logic, but I'm not totally sure _why_ it's meant to work this way. Maybe we only expected this type of file in our error payloads at some point?

Reply:
I'll leave it for now and follow-up with the product team about it as a follow-up.